config/defaults.config.php

```diff
@@ -135,6 +135,10 @@
 		// Disables the check if a mail address is in a valid format and can be resolved via DNS.
 		'disable_email_validation' => false,
 
+		// disable_mentions_removal (Boolean)
+		// Disables the automatic removal of mentions in ActivityPub postings.
+		'disable_mentions_removal' => false,
```

Comment:
Please mention the fact that we are only removing implicit mentions in the config name or doc or both.